apps/crn-server/src/controllers/calendars.ts

```diff
@@ -1,21 +1,16 @@
 import { NotFoundError } from '@asap-hub/errors';
 import {
+  CalendarController,
   CalendarDataObject,
+  CalendarDataProvider,
   CalendarResponse,
   CalendarUpdateRequest,
   ListCalendarResponse,
 } from '@asap-hub/model';
-import {
-  CalendarController,
-  CalendarDataProvider,
-} from '@asap-hub/server-common';
 
 export default class Calendars implements CalendarController {
-  dataProvider: CalendarDataProvider;
+  constructor(private dataProvider: CalendarDataProvider) {}
```

Comment:
does this do the same thing? never seen it before 😄

Reply:
It does the same thing, @peterstarling showed it to me.